docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -218,7 +221,20 @@ tcld login
 tcld apikey delete --id <api-key-id>
 ```
 
-## Manage API Keys for Service Accounts {#serviceaccount-api-keys}
+### Rotate an API key
+
+Temporal API keys automatically expire.
+Their maximum life is 90 days.
+Follow these steps to rotate API keys:
+
+1. Create a new key.
+   You may reuse key names if that helps.
+1. Ensure both the original key and new key are functioning properly before moving to the next step.
+1. Switch clients to load the new key and start using it.
+   Phase out the old key until is no longer being used.
```

Comment:
"Phase out the old key until is no longer being used" sounds odd to me. 

Consider rephrasing by combining with the next point. For example, "You may delete the old key once it is no longer in use."

Reply:
Adopting suggestion.

Reply:
```
1. Create a new key.
   You may reuse key names if that helps.
1. Ensure both the original key and new key are functioning properly before moving to the next step.
1. Switch clients to load the new key and start using it.
1. Delete the old key after it is no longer in use.
``